cli/src/commands/create-expo-stack.ts

```diff
@@ -316,15 +309,23 @@ const command: GluegunCommand = {
         const generateRerunScript = (cliResults: CliResults) => {
           let script = `npx create-expo-stack ${cliResults.projectName} `;
 
-          if (cliResults.packages.length > 0 && cliResults.packages[0].name === 'nativewindui') {
+          const nativewindUISelected = cliResults.packages.find((p) => p.name === 'nativewindui');
+
+          const navigationSelected = cliResults.packages.find((p) => p.type === 'navigation');
+
+          if (cliResults.packages.length > 0 && nativewindUISelected) {
             script += '--nativewindui ';
-            if (cliResults.packages[0].options.selectedComponents.length === 0) {
+
+            if (nativewindUISelected.options.selectedComponents.length === 0) {
               script += '--blank ';
+            } else if (nativewindUISelected.options.selectedComponents.length !== nativewindUIOptions.length) {
```

Comment:
just to confirm, in the event that the user selects all of the NativeWindUI components, they would just see a re-run script of `npx create-expo-stack@latest --nativewindui`, correct?

Versus if they select at least 1 component but less than all of the components, in which case they'll see `npx create-expo-stack@latest --nativewindui --selected-components={...}`?

Reply:
Yes what I aimed for. If you have everything selected it uses the default selection (everything). 

If you have unselected anything i.e more than 1 but less than the total number of options then we give them the list of components that were selected.

We could just always provide the list but it seems cleaner to not include the massive list when we don't need to